module.exports = {
    ApplicationError: 'Something went wrong. Please try again',
    BadRequestError: 'bad request',
    BankDeclineError: 'Bank decline transaction',
    DevAlreadyExistError: 'Dev with this login already exist',
    NotEnoughMoney: 'Not enough money',
    NotUniqueEmail: 'this email were already exist',
    RightsError: 'not enough rights',
    ServerError: 'server error',
    TokenError: 'token error',
    UncorrectPassword: 'incorrect password',
    UserNotFoundError: 'user with email not found',
    Transaction: `Problem with transaction creating`,
    USER_CREATE:'server error on user creation',
    USER_UPDATE:'cannot update user',
    USER_DIDNT_FIND: 'user with this data didn`t exist',
    USERS_DIDNT_FIND: 'problem with users downloading',
    WRONG_PASSWORD:'Wrong password',
    MONEY_TRANSACTION: 'server error on transaction creation',
    RATING_UPDATE: 'cannot update mark on this offer',
    RATING_MARK: 'cannot mark offer',
    RATING_FIND: 'problem with offer marking',
    EVENT_CREATE:`server error on Event creation`,
    EVENT_DELETE:'server error on delete operation',
    EVENT_FIND: `Problem during events find`,
    CONTEST_CREATE:'cannot create contests',
    CONTEST_UPDATE:'cannot update Contest',
    CONTEST_FIND:`Contests with this data didn't exist`,
    OFFER_FIND:`Offer with this data doesn't exist`,
    OFFER_UPDATE:'cannot update offer!',
    OFFER_CREATE:'cannot create offer',
    OFFER_FIND_PROBLEM:`Problem during finding offers`,
    CHAT_CREATE: (modelName) => `server error on ${modelName} creation`,
    CHAT_UPDATE: (modelName) => `cannot update ${modelName}`,
    CHAT_FIND: (modelName) => `${modelName} with this data didn't exist`,
    CHAT_CATALOG_DELETE: 'server error during delete catalog',
    CHAT_CONVERSATION_CREATE: 'cannot create/find conversation',
    BANK_DECLINE: 'Bank decline transaction',
    DATA_FOR_CONTEST: 'cannot get contest preferences',
    GET_CONTEST_BY_ID: 'problem with contest loading'
};